site/en/guides/thread-primer/node-roles-and-types.md

```diff
@@ -151,9 +151,200 @@ What you learned:
 *   A Border Router is used to connect Thread and non-Thread networks
 *   A Thread network might be composed of multiple partitions
 
+## Check your understanding
+
+<div>
+  <devsite-multiple-choice>
+    <div>A Thread network device can fulfill one of two forwarding roles. 
+    What are they?</div>
+    <div>
+      <div>Child Node.</div>
+      <div>Incorrect.</div>
+    </div>
+    <div correct>
+      <div>Router.</div>
+      <div>Correct.</div>
+    </div>
+    <div correct>
+      <div>End Device.</div>
+      <div>Correct.</div>
+    </div>
+    <div>
+      <div>Gateway.</div>
+      <div>Incorrect.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+<div>
+  <devsite-multiple-choice>
+    <div>What are the two primary types of Thread device?</div>
+    <div correct>
+      <div>Minimal Thread Device (MTD).</div>
+      <div>Correct.</div>
+    </div>
+    <div correct>
+      <div>Full Thread Device (FTD).</div>
+      <div>Correct.</div>
+    </div>
+    <div>
+      <div>Miniscule Thread Device (MTD).</div>
+      <div>Incorrect.</div>
+    </div>
+    <div>
+      <div>Sleepy End Device (SED).</div>
+      <div>Incorrect.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+
+<div>
+  <devsite-multiple-choice>
+    <div>Which of the following statements about Routers are incorrect?</div>
```

Comment:
I found it confusing to ask what's incorrect and have wrong answers say correct in the revealed text.

Can we say "Which of the following statements about Routers is not true?" and instead of "Correct" in the answer reveals say "This is true."

Reply:
Good catch - my oversight. And I like your suggestion.